Release 1.12.0 (#3948)
redhat-developer/vscode-openshift-tools: CHANGELOG.md

# Change Log

## 1.12.0 (February 28, 2024)
 * [#3880](https://github.com/redhat-developer/vscode-openshift-tools/pull/3880) Do not call getAllComponents() in activate.
 * [#3112](https://github.com/redhat-developer/vscode-openshift-tools/issues/3112) Extension takes several minutes to start if the current `~/.kube/config` context is an inaccessible cluster.
 * [#3931](https://github.com/redhat-developer/vscode-openshift-tools/issues/3931) New Serverless Function is not shown after creation.
 * [#3928](https://github.com/redhat-developer/vscode-openshift-tools/issues/3928) Can't load Helm chart.

>  * [#3879](https://github.com/redhat-developer/vscode-openshift-tools/pull/3879) When turned to using service account for a Sandbox a wrong warning me….

Reviewer: ```suggestion
 * [#3879](https://github.com/redhat-developer/vscode-openshift-tools/pull/3879) When turned to use a service account for a Sandbox a wrong warning message appears.
```

Author: Done
Plus  one  more commit title is extracted:

```
- * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project....
+ * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project when new project name is the same as the one set in the current-context.
```

>  * [#3848](https://github.com/redhat-developer/vscode-openshift-tools/pull/3848) Stop trying to connect to a cluster when it doesn't appear to be avai….

Reviewer: ```suggestion
 * [#3848](https://github.com/redhat-developer/vscode-openshift-tools/pull/3848) Stop trying to connect to a cluster when it doesn't appear to be available.
```

Author: Done

>  * [#3841](https://github.com/redhat-developer/vscode-openshift-tools/pull/3841) Update func to 1.13.0.

Reviewer: ```suggestion
 * [#3841](https://github.com/redhat-developer/vscode-openshift-tools/pull/3841) Update func CLI to 1.13.0.
```

Author: Done

>  * [#3838](https://github.com/redhat-developer/vscode-openshift-tools/issues/3838) Developer Sandbox Login experience enhancement.

Reviewer: ```suggestion
 * [#3838](https://github.com/redhat-developer/vscode-openshift-tools/issues/3838) New and Improved sign-in for Developer Sandbox Login experience.
```

Author: Done

>  * [#3822](https://github.com/redhat-developer/vscode-openshift-tools/issues/3822) Context menu item to scale Deployments.

Reviewer: ```suggestion
 * [#3822](https://github.com/redhat-developer/vscode-openshift-tools/issues/3822) Add Context menu item to scale OpenShift Deployments.
```

Author: Done

 * [#3814](https://github.com/redhat-developer/vscode-openshift-tools/issues/3814) "Open in Console" context menu item appears for non-OpenShift clusters.
 * [#3813](https://github.com/redhat-developer/vscode-openshift-tools/pull/3813) Force refresh App explorer on create project....
 * [#3812](https://github.com/redhat-developer/vscode-openshift-tools/issues/3812) As a user I initially couldn't find "Change Active Project".
 * [#3804](https://github.com/redhat-developer/vscode-openshift-tools/pull/3804) Improve switch context workflow.

>  * [#3665](https://github.com/redhat-developer/vscode-openshift-tools/issues/3665) Investigate replacing `Odo.isPodmanPresent`.

Reviewer: ```suggestion
 * [#3665](https://github.com/redhat-developer/vscode-openshift-tools/issues/3665) Enhance dev on podman experience
```

Author: Done

 * [#3640](https://github.com/redhat-developer/vscode-openshift-tools/issues/3640) Show context menu to Open in Terminal for the function created.
 * [#3638](https://github.com/redhat-developer/vscode-openshift-tools/issues/3638) Show vscode notification message if docker is not started for Fn commands.
 * [#3512](https://github.com/redhat-developer/vscode-openshift-tools/issues/3512) Add paste to the OpenShift terminal.

>  * [#3405](https://github.com/redhat-developer/vscode-openshift-tools/issues/3405) podman installed but propose to install podman.

Reviewer: ```suggestion
 * [#3405](https://github.com/redhat-developer/vscode-openshift-tools/issues/3405) Fix podman detection prompt
```
